### PR TITLE
adding accessibilityLabelledBy to documentation

### PR DESCRIPTION
**Pull request by fabriziobertoglio1987**
fixes https://github.com/facebook/react-native/issues/30846 
was build by https://github.com/grgr-dkrk with PR https://github.com/facebook/react-native/pull/32470
Related https://github.com/facebook/react-native/pull/34371

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Nice catch, thank you for the updates @fabriziobertoglio1987! 🙏 

Since this change has landed in `0.68` can you backport the addition to the versioned docs for `0.68` and `0.69` located in  https://github.com/facebook/react-native-website/tree/main/website/versioned_docs?

**Comment by fabriziobertoglio1987** [comment]
@Simek Thanks. I'll add it to 0.68 next week. I have a couple of high-priority tasks now.

File: docs/accessibility.md [hunk 1]
```diff
@@ -46,6 +46,23 @@ To use, set the `accessibilityLabel` property to a custom string on your View, T
 
 In the above example, the `accessibilityLabel` on the TouchableOpacity element would default to "Press me!". The label is constructed by concatenating all Text node children separated by spaces.
 
+### `accessibilityLabelledBy`
+
+A reference to another element nativeID used to build complex forms.
+The value of the `accessibilityLabelledBy` should match the `nativeID` of the other element:
```
Comment (huntie) [inline docs/accessibility.md:52]: Small nits.

```suggestion
The value of `accessibilityLabelledBy` should match the `nativeID` of the related element:
```

File: docs/accessibility.md [hunk 2]
```diff
@@ -46,6 +46,23 @@ To use, set the `accessibilityLabel` property to a custom string on your View, T
 
 In the above example, the `accessibilityLabel` on the TouchableOpacity element would default to "Press me!". The label is constructed by concatenating all Text node children separated by spaces.
 
+### `accessibilityLabelledBy`
+
+A reference to another element nativeID used to build complex forms.
```
Comment (huntie) [inline docs/accessibility.md:51]: Can we mark this as Android only, and perhaps include a docs link for `nativeID`? (**Note**: Suggestion below might not be the correct docs reference)

```suggestion
### `accessibilityLabelledBy` <div class="label android">Android</div>

A reference to another element [`nativeID`](/docs/view#nativeid) used to build complex forms.
```

File: docs/accessibility.md (hunk 1 above)
Comment (fabriziobertoglio1987) [reply docs/accessibility.md]: @huntie Thanks, Alex. I added the changes with commit https://github.com/facebook/react-native-website/commit/487a9259914b848c0f492a6f31c09a7d2d358f06

File: docs/accessibility.md (hunk 2 above)
Comment (fabriziobertoglio1987) [reply docs/accessibility.md]: @huntie Thanks, Alex. I added the changes with commit https://github.com/facebook/react-native-website/commit/487a9259914b848c0f492a6f31c09a7d2d358f06

File: docs/accessibility.md [hunk 3]
```diff
@@ -46,6 +46,23 @@ To use, set the `accessibilityLabel` property to a custom string on your View, T
 
 In the above example, the `accessibilityLabel` on the TouchableOpacity element would default to "Press me!". The label is constructed by concatenating all Text node children separated by spaces.
 
+### `accessibilityLabelledBy` <div class="label android">Android</div>
+
+A reference to another element [nativeID](view#nativeid) used to build complex forms.
```
Comment (huntie) [inline docs/accessibility.md:51]: Small change, can we reference the Markdown file with extension? This helps a little with maintainability as these references can easily be searched for and Docusaurus should find and substitute the correct URL if it changes in future :).

(We use a mixed pattern currently but this approach is better than the "from root" `/docs/view#nativeid` approach I suggested originally.)

```suggestion
A reference to another element [nativeID](view.md#nativeid) used to build complex forms.
```

**Review by huntie — COMMENTED** [review]
Thanks @fabriziobertoglio1987! Just one more thing I noticed :)

File: docs/accessibility.md (hunk 3 above)
Comment (fabriziobertoglio1987) [reply docs/accessibility.md]: Thanks a lot https://github.com/facebook/react-native-website/pull/3249/commits/769e6d0b153ed1b573a68c90fae5a424919a11cd @huntie

**Pull request merged**